front-end/package-lock.json

```diff
@@ -8470,218 +8470,29 @@
       }
     },
     "jest-watch-typeahead": {
-      "version": "1.1.0",
-      "resolved": "https://registry.npmjs.org/jest-watch-typeahead/-/jest-watch-typeahead-1.1.0.tgz",
-      "integrity": "sha512-Va5nLSJTN7YFtC2jd+7wsoe1pNe5K4ShLux/E5iHEwlB9AxaxmggY7to9KUqKojhaJw3aXqt5WAb4jGPOolpEw==",
+      "version": "0.6.5",
```

Comment:
Why is jest-watch-typeahead switching from 1.1.0 to 0.6.5? Also, latest version of jest-watch-typeahead is 2.2.2... so it is even further back from current version.

Reply:
Not 100% sure, but I went ahead and installed the latest version and pushed the changes.

Reply:
I updated Jest to the most recent version and that line of code stayed the same. All the tests pass, and everything functions as it should. If there's a way and a reason for me to update that particular item, please let me know!